addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -324,6 +332,12 @@
                                                 <t t-esc="record.attachment_number.raw_value"/>
                                             </span>
                                         </a>
+                                        <div style="padding-left:7%;padding-bottom:5%">
```

Comment:
We usually avoid having inline style. Could you extract it and/or use a generic class that could apply to other kanban view ? Maybe check with css-wise guys like @qsm-odoo .